website/components/LandingPage/CodeExample.tsx

```diff
@@ -1,50 +1,50 @@
 import { useSnapshot } from "valtio";
 import Highlight, { defaultProps } from "prism-react-renderer";
 import { GettingStarted } from "./GettingStarted";
-import { animationState, incDuration, decDuration } from "./animationState";
+import { state, incDuration, decDuration } from "./state";
 
 const exampleCode = (duration: number, count: number) => `
-  const animationState = proxy({
-    durationInSec: ${duration},
+  const state = proxy({
+    dur: ${duration},
     count: ${count}
   });
-  const incDuration = () => {
-    ++animationState.durationInSec;
-  };
-  const decDuration = () => {
-    --animationState.durationInSec;
+  const incDur = () => ++state.dur;
+  const decDur = () => --state.dur;
```

Comment:
```suggestion
  const incDur = () => { ++state.dur; };
  const decDur = () => { --state.dur; };
```
just in case, people get in a pitfall.